rfc6962-bis.xml

```diff
@@ -1175,7 +1175,7 @@ entry specified by <spanx style="verb">start</spanx>.
             A TLS extension (Section 7.4.1.4 of <xref target='RFC5246'/>) with type <spanx style="verb">transparency_info</spanx> (see <xref target="tls_transinfo_extension"/>). This mechanism allows TLS servers to participate in CT without the cooperation of CAs, unlike the other two mechanisms. It also allows SCTs and inclusion proofs to be updated on the fly.
           </t>
           <t>
-            An <xref target='RFC6960'>Online Certificate Status Protocol (OCSP)</xref> response extension (see <xref target="ocsp_transinfo_extension"/>), where the OCSP response is provided in the <spanx style="verb">certificate_status</spanx> TLS extension (Section 8 of <xref target='RFC6066'/>), also known as OCSP stapling. This mechanism is already widely (but not universally) implemented. It also allows SCTs and inclusion proofs to be updated on the fly.
+            An <xref target='RFC6960'>Online Certificate Status Protocol (OCSP)</xref> response extension (see <xref target="ocsp_transinfo_extension"/>), where the OCSP response is provided in the <spanx style="verb">CertificateStatus</spanx> message, provided that the TLS client included the <spanx style="verb">status_request</spanx> extension in the (extended) <spanx style="verb">ClientHello</spanx> (Section 8 of <xref target='RFC6066'/>). This mechanism, popularly known as OCSP stapling, is already widely (but not universally) implemented. It also allows SCTs and inclusion proofs to be updated on the fly.
```

Comment:
It requires the cooperation of the CA, but it also requires the server to fetch the updated OCSP response "on the fly" and to start stapling it in new handshakes.  Since the CA can't force the server to fetch the updated OCSP response, I think I'd prefer to append "with the cooperation of the CA".  Does that work for you?